ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -547,6 +586,61 @@ def test_nginx_commits_after_remove(tmpdir, settings, staticsite_path,
     manager().restart.assert_not_called()
 
 
+@pytest.mark.usefixtures('db')
+def test_mediacenter_installs_zippedmedia(tmpdir, settings, zippedmedia_path):
+    from ideascube.serveradmin.catalog import MediaCenter, ZippedMedia
+
+    settings.CATALOG_MEDIACENTER_INSTALL_DIR = tmpdir.strpath
+    MediaCenter.root = os.path.join(settings.STORAGE_ROOT, 'mediacenter')
+
+    p = ZippedMedia('test-media', {
+        'url': 'https://foo.fr/test-media.zip'})
+    h = MediaCenter()
+    h.install(p, zippedmedia_path.strpath)
+
+    root = tmpdir.join('test-media')
+    assert root.check(dir=True)
+
+    manifest = root.join('manifest.yml')
+    assert manifest.exists()
+
+    assert Document.objects.count() == 3
+    video = Document.objects.get(title='my video')
+    assert video.summary == 'my video summary'
+    assert video.kind == Document.VIDEO
+    assert Document.objects.search('summary').count() == 3
+
+    packaged_documents = list(Document.objects.filter(package_id='test-media'))
+    assert len(packaged_documents) == 3
+
+    # Be sure that referenced documents are the ones installed by the package
```

Comment:
In this test I do nothing but a check that documents are installed correctly.
I do not take any action to "make sure" or "ensure" that every thing is correct, I just want to "be sure" that this is correct.

However, maybe my English is not good enough and "make sure" is the right expression.
And it is in the comment so, I don't especially care :)